patches/@typescript-eslint+utils+8.0.0-alpha.41.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@typescript-eslint/utils/dist/ts-eslint/RuleTester.d.ts b/node_modules/@typescript-eslint/utils/dist/ts-eslint/RuleTester.d.ts
+index ee1176f..7868f50 100644
+--- a/node_modules/@typescript-eslint/utils/dist/ts-eslint/RuleTester.d.ts
++++ b/node_modules/@typescript-eslint/utils/dist/ts-eslint/RuleTester.d.ts
+@@ -113,7 +113,7 @@ interface RunTests<MessageIds extends string, Options extends readonly unknown[]
+     readonly invalid: readonly InvalidTestCase<MessageIds, Options>[];
+ }
+ interface RuleTesterConfig extends ClassicConfig.Config {
+-    readonly parser: string;
++    readonly parser?: string;
+     readonly parserOptions?: Readonly<ParserOptions>;
+ }
```

Comment:
@JoshuaKGoldberg Is the `parser` really required here? It seems that `typescript-eslint` will use the `@typecript-eslint/parser` from the peer dependency (if not explicitly installed) by default.

Reply:
Hey good timing, it's not! I _just_ released an update to the v8 beta that switches the `RuleTestBeta` that makes the `parser` optional: https://github.com/typescript-eslint/typescript-eslint/pull/9603.

I'll note that that's actually a breaking change to switch the tester from legacy config to flat config, so the tests in this plugin will need to:

* Move any custom `parser` or `parserOptions` inside a `languageOptions`
* Switch from `require.resolve('@babel/eslint-parser')` to `require('@babel/eslint-parser')`

It should be a manual find-and-replace. The bright side is, you're right that the typescript-eslint parser is now the default.

Reply:
https://deploy-preview-9627--typescript-eslint.netlify.app/blog/announcing-typescript-eslint-v8-beta#flat-configuration-ruletester